now.json

```diff
@@ -1,10 +1,17 @@
 {
+  "version": 2,
   "name": "vuetifyjs.com",
   "alias": "vuetifyjs.com",
+  "builds": [
```

Comment:
Is there any documentation on this option? We need it to run `yarn` from the root directory before building so the whole monorepo thing actually works properly.

Reply:
I tried making a `server.js` at the root instead with just `module.exports = require('./packages/docs/server')` but that didn't work either.
Adding a `functions` key to the config to use `includeFiles` gives the error `The property `builds` is not allowed in now.json when using Now 2.0 – please remove it.`
Trying to test locally with `now dev -d` so I can see what's going on without waiting for now to start building tries to type-check every ts file in the repo even though it's already been built by webpack, then infinitely loops back to building when it's done.

Reply:
Hi @KaelWD 

This PR was submitted awhile ago while `builds` was still documented. The new `functions` property has mostly replaced it. In this case, the `./packages/docs` website relies on build output from the other directories (I think) so I had to use `builds` instead of moving `./now.json` to `./packages/docs/now.json`. You can't mix `builds` and `functions`.